Packs/ArcherRSA/Integrations/ArcherV2/ArcherV2.py

```diff
@@ -537,6 +537,12 @@ def try_rest_request(
         Returns:
             requets.Response: the response object
         """
+        if method.casefold() == "post":
+            demisto.debug(f"Sending POST request to endpoint: {url_suffix} with data: {data}")
+
+        if method.casefold() == "get":
+            demisto.debug(f"Sending GET request to endpoint: {url_suffix} with params: {params}")
+
```

Comment:
we can consider using one liner 
demisto.debug(f"Sending {method.casefold()} request to endpoint: {url_suffix} with {params=}, {data=} ")

Reply:
Updated to one line